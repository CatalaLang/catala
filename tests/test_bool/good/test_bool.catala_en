## Article

```catala
declaration scope TestBool:
  context foo content boolean
  context bar content integer

scope TestBool:
  definition bar equals 1
  definition foo under condition bar >= 0 consequence equals true
  definition foo under condition bar < 0 consequence equals false
```
