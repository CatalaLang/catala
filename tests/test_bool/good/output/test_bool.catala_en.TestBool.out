[RESULT] Computation successful! Results:
[RESULT] bar = 1
[RESULT] foo = true
